eng/pipelines/common/xplat-setup.yml

```diff
@@ -30,6 +30,21 @@ jobs:
       - name: System.DisableZipDownload
         value: true
 
+      - ${{ if eq(parameters.jobParameters.buildConfig, 'checked') }}:
```

Comment:
I added these changes on my PR as well, if you want I can either remove them from there or you can just ingest the few changes from my PR you are missing here and I'll go ahead and close mine